### DISCUSSION TITLE
Ничего не понятно

### DISCUSSION THREAD

**Discussion by lynrayy** [hufrea/byedpi#157] [General]
Как настроить также как на компе? Не работает на телефоне, хоть бы посоветовали настройки какие ставить для операторов, ибо из коробки - не пашет. Пойдут аргументы от гудбай дипиай?

Йота

**Comment by lynrayy** [comment]
Зачем вообще в днс указан 1111, который у нас заблокирован? Бред же, ну

**Comment by MrRanMao** [comment]
У нас 1.1.1.1 не заблокирован. Если ваш провайдер блокирует сторонние подключения по протоколу dns, указывайте их сервера выданные по договору или используйте DoH. Очень сомневаюсь что сотовые операторы начали блокировать единично dns.
Какие у вас настройки стоят на ПК?

**Comment by lynrayy** [comment]
На пк те что сейчас для ютуба в последней версии гудбай дипиай

**Comment by MrRanMao** [comment]
> На пк те что сейчас для ютуба в последней версии гудбай дипиай

goodbyedpi и byedpi разные инструменты.
Приложение ByeDPI использует ciadpi для фрагментации трафика.

Последняя конфигурация по инструкции использует аргумент -9 в goodbye
В документации есть информация какие методы этот аргумент реализует:
```
 -9          -f 2 -e 2 --wrong-seq --wrong-chksum --reverse-frag --max-payload -q (this is the default)
 -f <value>  set HTTP fragmentation to value
 -e <value>  set HTTPS fragmentation to value
 --wrong-seq              activate Fake Request Mode and send it with TCP SEQ/ACK in the past.
 --reverse-frag           fragment (split) the packets just as --native-frag, but send them in the reversed order.
```

Если вы хотите такую-же конфигурацию на byedpi, почитайте документацию к данному инструменту и выставите аргументы уже для него.
 
 Учтите, что данная конфигурация подходит не для всех провайдеров и сотовых операторов.
 Для каждого оператора и региона надо находить отдельно. В таблице так же на странице у [goodbyedpi](https://github.com/ValdikSS/GoodbyeDPI/issues/378) есть перечень операторов, регионы и какие настройки в теории могут помочь. Но в виду постоянно меняющихся обстоятельств и настроек ТСПУ, они могут быть уже не актуальны.

**Comment by lynrayy** [comment]
Мне вот такое надо

`-9 --fake-gen 5 --fake-from-hex 160301FFFF01FFFFFF0303594F5552204144564552544953454D454E542048455245202D202431302F6D6F000000000009000000050003000000`

**Comment by lynrayy** [comment]
> почитайте документацию

Не хочу, как и другие люди

**Comment by MrRanMao** [comment]
> > почитайте документацию
> 
> Не хочу, как и другие люди

А придется, потому что конфигурация которую вы прислали выше, может не сработать на вашем сотовом операторе. Потому что настройки ТСПУ точно так же, как и мы все меняют.

**Comment by zimbabwe303** [comment]
Попробуйте вот эти варианты:
```
-s1 -Ar -o1 -At -f-1 -r1+s
-s1 -Ar -d1 -At -f-1 -r1+s
-s1 -At -d1 -At -f-1 -r1+s
-d1 -At -f-1 -r1+s
```
И вообще заглядывайте в [Discussions версии для Android](https://github.com/dovecoteescapee/ByeDPIAndroid/discussions/), там активнее всего, как мне кажется, обсуждение идёт по конкретным случаям.

**Comment by koreshforever** [comment]
> На пк те что сейчас для ютуба в последней версии гудбай дипиай

Мало того что вы написали issue в теме не мобильного приложения, перепутали два совершенно разных инструмента, ещё и не собираетесь ни в чём разбираться. 

Мне кажется тему можно закрывать, дальше как и в теме по андроид приложению всё замусорят писаниной и скриншотами...

**Comment by Damienbro** [comment]
> Попробуйте вот эти варианты:
> 
> ```
> -s1 -Ar -o1 -At -f-1 -r1+s
> -s1 -Ar -d1 -At -f-1 -r1+s
> -s1 -At -d1 -At -f-1 -r1+s
> -d1 -At -f-1 -r1+s
> ```
> 
> И вообще заглядывайте в [Discussions версии для Android](https://github.com/dovecoteescapee/ByeDPIAndroid/discussions/), там активнее всего, как мне кажется, обсуждение идёт по конкретным случаям.

Спасибо

**Comment by koreshforever** [comment]
> > Попробуйте вот эти варианты:
> > ```
> > -s1 -Ar -o1 -At -f-1 -r1+s
> > -s1 -Ar -d1 -At -f-1 -r1+s
> > -s1 -At -d1 -At -f-1 -r1+s
> > -d1 -At -f-1 -r1+s
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > И вообще заглядывайте в [Discussions версии для Android](https://github.com/dovecoteescapee/ByeDPIAndroid/discussions/), там активнее всего, как мне кажется, обсуждение идёт по конкретным случаям.
> 
> Спасибо

С таким же успехом можно приложить подорожник. Нужно хотябы запустить отладчик в браузере и смотреть проходят ли ответы и как быстро.

**Comment by zimbabwe303** [comment]
Явно требуется хоть какая-то пошаговая инструкция для простых людей, как подобрать параметры. Понятно, что её написать - задача нетривиальная, но попытаться можно. Только не я, конечно, - я сам дуб.
